user/admin.py

```diff
@@ -1,5 +1,27 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.utils.translation import gettext as _
+
 from .models import User
 
-admin.site.register(User, UserAdmin)
+
+@admin.register(User)
+class UserAdmin(DjangoUserAdmin):
```

Comment:
The `UserAdmin` class is correctly registered with the Django admin site, allowing management of the custom `User` model.